src/pages/ReimbursementAccount/USD/BusinessInfo/subSteps/NameBusiness.tsx

```diff
@@ -20,7 +21,8 @@ function NameBusiness({onNext, onMove, isEditing}: SubStepProps) {
     const defaultCompanyName = reimbursementAccount?.achData?.companyName ?? '';
     const bankAccountID = reimbursementAccount?.achData?.bankAccountID;
 
-    const shouldDisableCompanyName = !!(bankAccountID && defaultCompanyName && reimbursementAccount?.achData?.state !== 'SETUP');
+    const bankAccountState = reimbursementAccount?.achData?.state ?? '';
+    const shouldDisableCompanyName = !!(bankAccountID && defaultCompanyName && ![BankAccount.STATE.VERIFYING, BankAccount.STATE.PENDING].includes(bankAccountState));
```

Comment:
This condition is now a bit different with the old one, what's the reason for that?